usaspending_api/download/v2/download_admin.py

```diff
@@ -0,0 +1,56 @@
+from django.conf import settings
+from django.db.models import Q
+
+from usaspending_api.common.sqs_helpers import get_sqs_queue_resource
+from usaspending_api.download.filestreaming import csv_generation
+from usaspending_api.download.lookups import JOB_STATUS_DICT
+from usaspending_api.download.models import DownloadJob
+
+
+class DownloadAdministrator:
```

Comment:
This may be a good candidate to centralize some of our Data Access code/logic for a model (`DownloadJob` in this case) per the dev meeting topic here: https://github.com/fedspendingtransparency/data-act-documentation/blob/master/dev_meetings/WeeklyDevMeeting_2019-02-04.md#model-managers

Reply:
This is the starting block to refactoring the current Download logic in `views.py` and `csv_generation.py`. I thought about using a manager, but I was running out of time and it would only be a subset of the current class functionality